spec/requests/users_controller_spec.rb

```diff
@@ -5559,6 +5559,146 @@ def revoke(user, skip_remote: false)
     end
   end
 
+  describe "#user_menu_bookmarks" do
+    fab!(:post) { Fabricate(:post) }
+    fab!(:topic) { Fabricate(:post).topic }
+    fab!(:bookmark_with_reminder) { Fabricate(:bookmark, user: user, bookmarkable: post) }
+    fab!(:bookmark_without_reminder) { Fabricate(:bookmark, user: user, bookmarkable: topic) }
+
+    before do
+      TopicUser.change(user.id, post.topic.id, total_msecs_viewed: 1)
+      TopicUser.change(user.id, topic.id, total_msecs_viewed: 1)
+      BookmarkReminderNotificationHandler
+        .new(bookmark_with_reminder)
+        .send_notification
+    end
+
+    context "when logged out" do
+      it "responds with 404" do
+        get "/u/#{user.username}/user-menu-bookmarks"
+        expect(response.status).to eq(404)
+
+        get "/u/#{admin.username}/user-menu-bookmarks"
+        expect(response.status).to eq(404)
```

Comment:
```suggestion
        get "/u/#{user.username}/user-menu-bookmarks"
        expect(response.status).to eq(404)
```

The additional assertion here doesn't really add anything meaningful.